SpatialGDK/Documentation/content/non-unreal-layers.md

```diff
@@ -1,25 +1,17 @@
+By default, the GDK for Unreal uses a single Unreal server-worker type to handle all server-side computation. However, you can  set up additional server-worker types that do not use Unreal or the GDK.
 
-You can use non-Unreal layers to modularize your game’s functionality so you can re-use the functionality across different games. For example, you could use a non-Unreal layer written in Python that interacts with a database or other 3rd party service, such as [Firebase](https://firebase.google.com/) or [PlayFab](https://playfab.com/).
+You can use these non-Unreal server-worker types to modularize your game’s functionality so you can re-use the functionality across different games. For example, you could use a non-Unreal server-worker type written in Python that interacts with a database or other third-party service, such as [Firebase](https://firebase.google.com/) or [PlayFab](https://playfab.com/).
 
-## Layers in SpatialOS
-A SpatialOS layer has two elements;
-
-* a group of SpatialOS [component definitions](https://docs.improbable.io/unreal/alpha/content/glossary#spatialos-component),
-* server-worker instances of a worker type that have [write access authority](https://docs.improbable.io/unreal/alpha/content/glossary#authority) over the group of components.
-
-(For information on layers in non-GDK SpatialOS development, see the [SpatialOS documentation](https://docs.improbable.io/reference/latest/shared/worker-configuration/layers#layers).)
-
-## How to integrate non-Unreal layers into your game
+## How to integrate non-Unreal server-worker types into your game
 In order to interact with each other, Unreal and non-Unreal server-worker instances need to send and receive updates to and [commands](https://docs.improbable.io/reference/latest/shared/glossary#command) for the same SpatialOS components. We recommend that you define the SpatialOS components used by non-Unreal worker instances manually in [schema](https://docs.improbable.io/unreal/alpha/content/glossary#schema) files, separate from those automatically generated by the GDK.  This is because:
 
 * SpatialOS command data in GDK-generated schema files is encoded as byte strings, making deserialization of commands in non-Unreal server-worker types more difficult.
 * It aids portability; you can more easily re-use any non-Unreal server-worker types when you have an accompanying schema file.
 
-Default single Unreal layer development doesn’t accommodate schema files that haven’t been generated by the SDK, so you need to set your game up to handle this.
+Default single Unreal server-worker type development doesn’t accommodate schema files that haven’t been generated by the SDK, so you need to set your game up to handle this.
```

Comment:
```suggestion
Default single Unreal server-worker type development doesn’t accommodate schema files that haven’t been generated by the GDK, so you need to set your game up to handle this.
```

not your work but can correct this at the same time?

Reply:
Oops! Will do